public/css/main.css

```diff
@@ -773,7 +773,7 @@ video#bgvid {
     background-color: white;
     width: 100%;
     height: 100%;
-    position: fixed;
+    position: relative;
```

Comment:
Renaming the ID doesn't work as mapbox.js seems to have an issue loading the map entirely if the ID is changed.

![image](https://user-images.githubusercontent.com/20068009/27440053-9005695c-5737-11e7-85a7-63fcfb27e8cd.png)